### DISCUSSION TITLE
How to control the animation to start only when the element comes into view

### DISCUSSION THREAD

**Discussion by weilun0510** [animate-css/animate.css#1586] [General]
How to control the animation to start only when the element comes into view

**Comment by warengonzaga** [comment]
You can do it by using JS, creating a function in JS that listens to an event if the element is visible, and then adding the animation classes to the element to animate it.

**Reply by weilun0510** [reply]
thanks

**Comment by zcx888841** [comment]
[wow.js](https://github.com/matthieua/WOW) can help you

**Reply by KodingKhurram** [reply]
> [wow.js](https://github.com/matthieua/WOW) can help you

Is there any free version of this? @zhaochengxiang , you can build such a thing quickly with JS.

See [this](https://github.com/animate-css/animate.css/discussions/1510)
